.gitignore

```diff
@@ -65,4 +65,4 @@ buildNumber.properties
 *.log
 credentials.cached
 *.lcs
-.tiles
\ No newline at end of file
+.tiles
```

Comment:
This is minor, but since we didn't actually change this file at all, there should also not be any whitespace changes